miden-crypto/src/hash/algebraic_sponge/rescue/arch/x86_64_avx2.rs

```diff
@@ -189,9 +189,8 @@ unsafe fn add_small(
 #[inline(always)]
 unsafe fn maybe_adj_sub(res_wrapped_s: __m256i, mask: __m256i) -> __m256i {
     // The subtraction is very unlikely to overflow so we're best off branching.
-    // The even u32s in `mask` are meaningless, so we want to ignore them. `_mm256_testz_pd`
-    // branches depending on the sign bit of double-precision (64-bit) floats. Bit cast `mask` to
-    // floating-point (this is free).
+    // `_mm256_testz_pd` branches depending on the sign bit of double-precision (64-bit) floats.
```

Comment:
The original comment mentioned that "even u32s in mask are meaningless" and explained why we use `_mm256_testz_pd`. This context might be helpful for future maintenance.

Was the removal intentional? The new comment is cleaner, but the removed explanation provided useful implementation details about why the bitcast works.